Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/omf/OmfRecord.java

```diff
@@ -38,6 +38,14 @@ public abstract class OmfRecord {
 	public final static byte LEXTDEF = (byte)0xB4;
 	public final static byte LPUBDEF = (byte)0xB6;
 	public final static byte LCOMDEF = (byte)0xB8;
+	public final static byte CEXTDEF = (byte)0xBC;
+	public final static byte COMDAT = (byte)0xC2;
+	public final static byte LINKSYM = (byte)0xC4;
```

Comment:
This should be `LINSYM`.

Reply:
There is also apparently an obsolete record: `8EH TYPDEF—Type Definition Record`

Reply:
And I think we are missing `B2H or B3H BAKPAT—Backpatch Record`